native/src/hooks/useLoadCityContent.ts

```diff
@@ -76,7 +75,14 @@ const useLoadCityContent = <T>({ cityCode, languageCode, load }: UseLoadCityCont
         }
       : null
 
-  return { ...getError(), loading, refresh, data }
+  return { error: getError(), loading, refresh, data }
 }
 
+// Simple utility helper to just load cities and languages
+export const useSimpleLoadCityContent = (params: {
```

Comment:
Will be in #822 since tunews is not using normal fetch logic but endless scroll/pagination fetch logic and therefore we still need cities and languages but not a `load` function. However, I might rewrite this such that we only need one export.